sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ContainerPropertiesTest.java

```diff
@@ -0,0 +1,140 @@
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.InternalObjectNode;
+import com.azure.cosmos.models.ComputedProperty;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.SqlQuerySpec;
+import com.azure.cosmos.rx.TestSuiteBase;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Ignore
```

Comment:
why are we ignoring it? Is the support not yet there for emulator?

Reply:
Yes, it's not there in emulator yet.